docs/notifications.md

```diff
@@ -0,0 +1,96 @@
+# Notifications
+
+### Notification Types
+
+#### Package Releases
+
+Private Packagist will notify you every time we find one or more new releases for one of your packages.
+Notification channels allow you to specify about which private packages you want to get notified and whether you also want to receive notifications for mirrored packages as well.
```

Comment:
This is a bit out of the blue, what's a notification channel? At least mention they are explained below?

Reply:
Remove `notification channels` to not confuse users when reading this